packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -293,7 +293,7 @@ export class List extends BaseComponent<IListProps, IListState> {
     let divProps = getNativeProps(this.props, divProperties);
 
     // assign list if no role
-    role = role || 'list';
+    role = (typeof role === 'undefined') ? 'list' : role;
```

Comment:
@dzearing because you could reassign `undefined` (it's not a keyword) in older versions of JS people tend to recommend the `typeof` approach, but you're right, ES5 made it readonly, so it's probably not a risk anymore